### DISCUSSION TITLE
i can't enter on all weeks

### DISCUSSION THREAD

**Discussion by MaximRubic** [JordanSantiagoYT/FNF-JS-Engine#566] [Q&A]
Uncaught Error: Null Object Reference
Paths.initSplash() [line 128]
NoteSplash.loadAnims() [line 116]
NoteSplash.setupNoteSplash() [line 63]
NoteSplash.new() [line 44]
PlayState.create() [line 1205]
flixel/FlxGame.switchState() [line 639]
flixel/FlxGame.update() [line 726]
flixel/FlxGame.step() [line 679]
flixel/FlxGame.onEnterFrame() [line 558]
openfl/events/EventDispatcher.__dispatchEvent() [line 426]
openfl/display/DisplayObject.__dispatch() [line 1398]
Please report this error to the GitHub page: https://github.com/JordanSantiagoYT/FNF-JS-Engine
The engine has saved a crash log inside the crash folder, If you're making a GitHub issue you might want to send that!

**Comment by JordanSantiagoYT** [comment]
You are missing a configuration file (.txt) for your note splash skin.

To fix this, go into the Note Splash editor and make your config file.
